app/models/concerns/oregon_digital/document_metadata.rb

```diff
@@ -202,7 +202,8 @@ module DocumentMetadata
         { name: :original_filename, section_name: '' },
         { name: :full_size_download_allowed, section_name: '' },
         { name: :date_modified, section_name: '' },
-        { name: :date_uploaded, section_name: '' }
+        { name: :date_uploaded, section_name: '' },
+        { name: :archival_object_id, section_name: 'Identifiers' }
```

Comment:
This list is for ordering terms on the form, and the section_name var is for the beginning of a new section of the form. Can you move this to the position you want it?

Reply:
total incomprehension